src/components/Inputs/WalletPassphrase.tsx

```diff
@@ -28,14 +28,24 @@ import InfoBox from '../InfoBox'
 import Input from './Input'
 
 interface Props {
-  value: string
-  onChange: (passphrase: string) => void
+  onPassphraseConfirmed: (passphrase: string) => void
+  setIsPassphraseConfirmed: (match: boolean) => void
   className?: string
 }
 
-const WalletPassphrase = ({ value, onChange, className }: Props) => {
+const WalletPassphrase = ({ onPassphraseConfirmed, setIsPassphraseConfirmed, className }: Props) => {
   const { t } = useTranslation('App')
   const [isConsentActive, setIsConsentActive] = useState(false)
+  const [value, setValue] = useState('')
+  const [confirmValue, setConfirmValue] = useState('')
+  const passphraseIsNotUsed = !value && !confirmValue
```

Comment:
Super ultra nit: I personally like to add an empty line between state declarations and logic :)

Reply:
done!